docs/about.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />
 
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>About Multi-Armed Bandits &mdash; MABWiser 2.2.0 documentation</title>
```

Comment:
Unrelated to this PR --but I just noticed that in the Citations section the URL hyperrefs does not show up properly.

https://fidelity.github.io/mabwiser/#citation

Can we plz fix that as well, now that we are touching the docs?